recipe/meta.yaml

```diff
@@ -104,7 +104,7 @@ source:
     - patches/0011-add-missing-headers.patch
 
 build:
-  number: 0
+  number: 1
   skip: true  # [cuda_compiler_version == "10.2"]
   # always crashes
   skip: true  # [win and cuda_compiler_version in ("11.0", "11.1")]
```

Comment:
These have all been dropped in conda-forge ( https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/1708 ). So we can drop the `skip`s here

```suggestion
```